packages/web-forms/src/components/widgets/MultiselectDropdown.vue

```diff
@@ -0,0 +1,87 @@
+<script lang="ts" setup>
+import type { SelectItem, SelectNode } from '@getodk/xforms-engine';
+import PrimeMultiSelect from 'primevue/multiselect';
+
+const props = defineProps<{ question: SelectNode, style?: string}>();
+defineEmits(['update:modelValue']);
+
+const setSelectNValue = (values: SelectItem[]) => {
+	for(const v of props.question.currentState.value){
+		props.question.deselect(v);
+	}
+	for(const v of values) {
+		props.question.select(v);
+	}
```

Comment:
Awkward! This seems like it's kind of forced by the `PrimeMultiSelect` API? Yeah, it seems like we definitely want a better engine API for this kind of thing.

Reply:
yeah, if `select` method of future `SelectMultipleNode` could take `SelectItem[]` as an argument that would be great.